Formula/conman.rb

```diff
@@ -0,0 +1,44 @@
+class Conman < Formula
+  desc "Serial console management program supporting a large number of devices"
+  homepage "https://github.com/dun/conman"
+  url "https://github.com/dun/conman/archive/conman-0.3.1.tar.gz"
+  sha256 "cd47d3d9a72579b470dd73d85cd3fec606fa5659c728ff3c1c57e970f4da72a2"
+  license "GPL-3.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "freeipmi"
+
+  def install
+    system "./bootstrap"
+    system "./configure", *std_configure_args, "--disable-silent-rules"
```

Comment:
```suggestion
    system "./configure", *std_configure_args, "--disable-silent-rules", "--sysconfdir=#{etc}"
```
Let's not install configuration files into the cellar. These will get blown away by `brew upgrade`.